google_calendar_clickable_where/clickable_where.user.js

```diff
@@ -57,34 +29,25 @@ w.clickable_where.make_all_links_clickable = function make_all_links_clickable (
                     g = n.textContent.match(exp);
                 }
             }
-            else if (n.tagName.toLowerCase() != 'a') {
-                w.clickable_where.make_all_links_clickable(n);
+            else if ((n.nodeType == n.ELEMENT_NODE) && (n.tagName.toLowerCase() != 'a')) {
+                window.make_child_links_clickable(n);
             }
         }
     }
 };
 
-if (typeof (opera) != "undefined") {
-    // Start the mainpart, if the site is loaded
-    window.addEventListener('DOMContentLoaded', w.clickable_where.main, true);
-} else {
-    w.clickable_where.main();
+function whereClickHandler(e) {
```

Comment:
also, this function should not be in the bare namespace, it should be underneath the clickable_where namespace.